rector_examples/path_alias_manager_updated.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+/**
+ * This demonstrates the deprecated static calls that might be called from procedural code like `.module` files.
+ */
+
+/**
+ * A simple example using the minimum number of arguments.
+ */
+function simple_example() {
+  \Drupal::service('path_alias.manager');
```

Comment:
Is there a convention for this? In my forthcoming blog post, I was recommending putting both examples "needs updated" and "desired updated" in the same file. Is the "_updated" PHP file.

Reply:
@nerdstein, I've been making pairs of files. The `_updated` or `Updated` ones are copy & paste and then run Drupal Rector to see the changes.

I'd like to gather feedback on the whole examples pattern. What is helpful, what isn't? Can we make it easier to develop / read?

I also have a task internally to look at automated testing of the Rector rules which may cover some of this, but isn't 100% overlapping.

Reply:
I'd appreciate two separate directories and some naming conventions tied to the specific rector rules.

Reply:
Thanks, @nerdstein.

I agree naming conventions would help. The two separate folders is an interesting one as well.

I think documenting naming conventions will make this easier. There are some currently that are mostly followed, but we should take another look.